src/reusecore/PageHeader/index.js

```diff
@@ -4,6 +4,7 @@ import {Link, graphql, useStaticQuery} from "gatsby";
 import slugify from "../../utils/slugify";
 import PageHeaderWrapper from "./pageHeader.style";
 import Image from "../../components/image";
+import layer5_img from "../../assets/images/layer5/layer5-only/svg/layer5-gray-no-trim.svg";
```

Comment:
Wow. I don't know whether to be impressed or to just expect this from 2020's MeshMate of the Year. With great precision, you selected the extact flavor of this logo - the svg, the gray, the no trim, the no tagline... 🎯

Reply:
Haha, it's been some time doing neat work 😎 like this!